lib/wrapbox/runner/ecs.rb

```diff
@@ -143,7 +146,23 @@ def run_cmd(cmds, container_definition_overrides: {}, **parameters)
             )
           end
         end
-        ths.each(&:join)
+        ths.each { |th| th&.join }
+
+        true
+      rescue SignalException => e
+        sig = "SIG#{Signal.signame(e.signo)}"
```

Comment:
[nit] How about checking that `ths` array is not empty?
Because the following log messages assume that the tasks are running, we will try to confirm even if they don't exist.

Reply:
I want to output `SIGINT` to log message.
But `Interruptl#to_s` is empty string.